Pagina de logueo

Lordland opened the pull request:
Añadida página principal para entrar como empresa o como particular.
Añadida lógica para comprobar usuarios de empresas en BD.
Añadida lógica para sacar precios de BD con un 10% de descuento aplicado
ya.

Con este pull queda terminada mi 2º historia y principio de la 3º. Falta por acordar con Paloma
detalles de la visualización de la tienda.

Lordland committed:
Pagina de logueo

Añadida página principal para entrar como empresa o como particular.
Añadida lógica para comprobar usuarios de empresas en BD.
Añadida lógica para sacar precios de BD con un 10% de descuento aplicado
ya.

thewillyhuman reviewed (approved):
Supongo que está todo probado y testado así que se aprueba.

ZenMaster91 commented:
Antes de proceder al merge puedes explicarme:
> Añadida lógica para sacar precios de BD con un 10% de descuento aplicado.

Según lo acordado con el cliente existirá un segundo precio de lista siendo este el correspondiente a las empresas.

thewillyhuman reviewed (changes requested):
Por favor acláreme el concepto del 10% de descuento.

Vodkaneitor93 commented:
10% de descuento es el descuento que juraría habíamos acordado para las empresas. En cuanto a lo del segundo precio de lista no comprendo a qué te refieres exactamente, según entendí si un particular entra se le muestran los productos sin precio modificado y si entra una empresa con su inicio de sesion, se le muestra el listado de productos con el precio resultante tras el descuento.

ZenMaster91 commented:
Vale si, entiendo la confusión. Sin embargo mejor utiliza el atributo `company_price` que hay en cada producto. **Pro Tip:** Si usas `SELECT * FROM myshop.full_products;` puedes ver todos los atributos juntos de cada producto en una sola tabla.

Vodkaneitor93 commented:
Vale, comprendido el concepto. Me dispongo a modificar. Gracias por la explicacion

Lordland committed:
Eliminada lógica para descuentos

Eliminado método para sacar productos con precio ya de descuento, el
modelo actualizado ahorrará esa consulta a BD
